### PR TITLE
Various CI fixes

### PR DESCRIPTION
**Pull request by arthuro555**
- Make `!update` work when the file has been renamed
 - Don't generate a PR until the extension passes automatic tests
 - Add/Update issue output messages for increased clarity for the users of what's going on

Workflow deployed and tested on arthuro555/GDevelop-extensions, you can test it there.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-    "typescript.tsdk": "node_modules/typescript/lib"
+    "typescript.tsdk": "node_modules/typescript/lib",
+    "yaml.schemas": {
```
Comment (4ian) [inline .vscode/settings.json:3]: Let's revert this maybe? You should be able to have this in your own settings?

**Review by 4ian — APPROVED** [review]
👍

File: .vscode/settings.json (hunk 1 above)
Comment (arthuro555) [reply .vscode/settings.json:3]: I've left this in here because I think everyone appreciates having autocomplete and validation, not just me :p

Comment (4ian) [reply .vscode/settings.json:3]: Sure but this is an absolute path on your computer ;)

Comment (arthuro555) [reply .vscode/settings.json:3]: oh 🤦 I'll remove it

**Pull request merged**